Public/Src/Utilities/Utilities/StringTable.cs

```diff
@@ -558,7 +558,10 @@ public StringId AddString(string value)
                 bufferNum++;
 
                 // Make sure we don't overflow the buffer we're indexing into
-                Contract.Assert(bufferNum < NumByteBuffers, $"Exceeded the number of ByteBuffers allowed in this StringTable: {bufferNum} >= {NumByteBuffers}");
+                if (bufferNum >= NumByteBuffers)
+                {
+                    Contract.Assert(false, $"Exceeded the number of ByteBuffers allowed in this StringTable: {bufferNum} >= {NumByteBuffers}");
+                }

```

Comment:
Unrelated changes, but can affect performance because we avoid creating interpolated string.